src/main/java/kata/makers/Application.java

```diff
@@ -0,0 +1,56 @@
+package kata.makers;
+
+import java.util.HashMap;
+import java.util.Scanner;
+import kata.makers.exception.CreateFlightException;
+import kata.makers.exception.CreateFlightReservationException;
+import kata.makers.service.BookingService;
+import kata.makers.service.FlightService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Application {
+
+  private static final Logger log = LoggerFactory.getLogger(Application.class);
+
+  public static void main(String[] args) {
+    FlightService flightService = new FlightService(new HashMap<>());
+    BookingService bookingService = new BookingService(flightService);
+
+    Scanner scanner = new Scanner(System.in);
+    while (true) {
```

Comment:
Is there a chance to make this loop less complex? There is while loop with nested if/else and try/catch as well

Reply:
I can wrap the try catch block with a function, so that when people can know what action will be triggered when entering different input